### PR TITLE
technical review of openshift docs

### PR DESCRIPTION
**Pull request by christinaausley**
Technical review of OpenShift doc references following merge of https://github.com/camunda/camunda-platform-docs/pull/1125.

### CODE REVIEW

File: docs/self-managed/platform-deployment/openshift-helm.md [hunk 1]
```diff
@@ -3,45 +3,47 @@ id: openshift-helm
...
+The `Elasticsearch`, `Keycloak`, and `Postgresql` charts all specify default non-root users for security purposes. To deploy these charts through the `camunda-platform` chart, these default values must be removed. Unfortunately, due to a [longstanding bug in Helm](https://github.com/helm/helm/issues/9136) affecting all Helm versions from 3.2.0 and greater, this makes the installation of the chart (when deploying any of these sub-charts) more complex.
+
+Note that this is only an issue if you are deploying `Elasticsearch`, `Keycloak` (via `Identity`), or `Postgresql` (via `Keycloak`). If you are not deploying these, or not via the `camunda-platform` chart, or you are using a [permissive SCC](./openshift.md#permissive-scc).
```
[the remark this answers is not in the document]
Comment (npepinpe) [reply docs/self-managed/platform-deployment/openshift-helm.md:26]: Yes, sorry, I must have cut off parts of it while restructuring things.

If you aren't doing any of these things, then this issue does not affect your deployment.

[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/self-managed/platform-deployment/openshift-helm.md:26]: Thank you @npepinpe ! Will adjust.

**Review by npepinpe — APPROVED** [review]
Looks great, thanks!

**Comment by akeller** [comment]
Thanks y'all 😄

**Pull request merged**